### PR TITLE
Benchmark is added to README resolves #10

### LINKED ISSUE #10: We need benchmark testing
On readme file we need to show the benefits of this library. Some benchmarking data must be published.

Maybe we can use this library for building benchmark tests: https://pypi.org/project/pytest-benchmark/

### PR DESCRIPTION
**Pull request by bilbeyt**

### CODE REVIEW

File: fast_pagination/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class DenemeConfig(AppConfig):
```
Comment (miratcan) [inline fast_pagination/apps.py]: What is deneme?

File: fast_pagination/management/commands/generate_users.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    def handle(self, *args, **options):
+        batch_size=1000
+        user_count = options["user_count"]
+        users = []
+        for i in range(1, user_count + 1):
+            username = f"user_{i}"
+            user = User(username=username)
+            user.set_unusable_password()
+            users.append(user)
+            if i%batch_size == 0:
```
Comment (miratcan) [inline fast_pagination/management/commands/generate_users.py]: pep8

File: fast_pagination/tests.py [hunk 3]
```diff
@@ -36,5 +41,38 @@ def test_key_not_found(self):
...
+        'NAME': 'paginator',
+        'USER': 'postgres',
+        'HOST': 'localhost',
+        'PORT': 5432,
+    }
+
+
+def speed_test(queryset, paginator_cls):
+    paginator = paginator_cls(queryset, 1000)
+    print(paginator.page_range)
```
Comment (miratcan) [inline fast_pagination/tests.py]: print?

**Review by miratcan — CHANGES_REQUESTED** [review]

File: fast_pagination/apps.py (hunk 1 above)
Comment (bilbeyt) [reply fast_pagination/apps.py]: Changed.

File: fast_pagination/management/commands/generate_users.py (hunk 2 above)
Comment (bilbeyt) [reply fast_pagination/management/commands/generate_users.py]: done

File: fast_pagination/tests.py (hunk 3 above)
Comment (bilbeyt) [reply fast_pagination/tests.py]: removed

**Review by miratcan — APPROVED** [review]

**Pull request merged**